session/session.go

```diff
@@ -112,16 +112,16 @@ func New(
 	}
 }
 
-// SetAvailableHTTPSBouncer sets the HTTP bouncer base URL.
-func (s *Session) SetAvailableHTTPSBouncer(baseURL string) {
+// AddAvailableHTTPSBouncer adds the HTTP bouncer base URL.
+func (s *Session) AddAvailableHTTPSBouncer(baseURL string) {
 	s.AvailableBouncers = append([]model.Service{}, model.Service{
 		Address: baseURL,
 		Type:    "https",
 	})
 }
 
-// SetAvailableHTTPSCollector adds an HTTP bouncer base URL.
-func (s *Session) SetAvailableHTTPSCollector(baseURL string) {
+// AddAvailableHTTPSCollector adds an HTTP bouncer base URL.
```

Comment:
@xhdix Like above, plus there's a typo: the comment should mention the collector rather than the bouncer

Reply:
@bassosimone done. thanks.